### DISCUSSION TITLE
NextJS route size exceeding 100% limit in Vercel

### DISCUSSION THREAD

**Discussion by omar-zahid** [trpc/trpc#5100] [General]
I'm currently enjoying working with TRPC but getting worried with the limit below.
Will this be a problem? I can't find any documentation related to this subject.
The size is marked "red" at 124%. I'm guessing the limit is 50MB (I'm on Vercel's Pro plan).

![image](https://github.com/trpc/trpc/assets/57479337/82fc9020-e608-4fa2-bd79-9a69824da087)

My application is still in development and I'm expecting the features to go up by 10-20x or even more.
Can you share your functions size. (Deployments -> Select production deployment -> Functions -> Select path [trpc])

**Comment by KATT** [comment]
https://github.com/trpc/trpc/tree/main/examples/soa
